components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -257,7 +257,6 @@ export class GitpodServerEEImpl<C extends GitpodClient, S extends GitpodServer>
                 originalWorkspaceId: workspaceId,
                 layoutData
             });
-            this.workspaceDb.trace({ span }).store(workspace);
```

Comment:
not sure where this comes from in the first place.
However, we've just pulled the workspace out of the database a few lines before, and don't modify it here. It doesn't make sense to write it back then. Looks like a leftover to me.